entity-service/src/test/kotlin/com/egm/stellio/entity/repository/Neo4jRepositoryTests.kt

```diff
@@ -46,10 +47,48 @@ class Neo4jRepositoryTests {
     @Autowired
     private lateinit var relationshipRepository: RelationshipRepository
 
+    @Autowired
+    private lateinit var partialEntityRepository: PartialEntityRepository
+
+    private val beekeeperUri = "urn:ngsi-ld:Beekeeper:1230".toUri()
+
+    @Test
+    fun `it should merge a partial entity to a normal entity`() {
+        val partialEntity = PartialEntity(beekeeperUri)
+        partialEntityRepository.save(partialEntity)
+        createEntity(beekeeperUri, listOf("Beekeeper"))
+
+        neo4jRepository.mergePartialWithNormalEntity(beekeeperUri)
+
+        assertFalse(partialEntityRepository.existsById(beekeeperUri))
+        assertTrue(entityRepository.existsById(beekeeperUri))
+        val beekeeper = entityRepository.findById(beekeeperUri)
+        assertEquals(listOf("Beekeeper"), beekeeper.get().type)
+
+        neo4jRepository.deleteEntity(beekeeperUri)
+    }
+
+    @Test
```

Comment:
What about properties of partial entity ? They will be merged too ?
Can we add a test for that ?

Reply:
partial entities do not have properties (nor relationships).